docs/python/README.rst

```diff
@@ -52,6 +52,9 @@ replaces *scikit-learn* to compute the predictions.
 Changes
 -------
 
+0.5.0
+^^^^^
```

Comment:
add a link to the release notes here (even if the link doesn't exist today - it will exist once the branch is created). see entry for 0.4.0 below.

Reply:
Added.